3.6/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# This image provides a Python 3.6 environment you can use to run your Python
+# applications.
+FROM centos/s2i-base-centos7
+
+MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
+
+EXPOSE 8080
+
+ENV PYTHON_VERSION=3.6 \
+    PATH=$HOME/.local/bin/:$PATH \
+    PYTHONUNBUFFERED=1 \
+    PYTHONIOENCODING=UTF-8 \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    PIP_NO_CACHE_DIR=off
+
+ENV SUMMARY="Platform for building and running Python $PYTHON_VERSION applications" \
+    DESCRIPTION="Python $PYTHON_VERSION available as docker container is a base platform for \
+building and running various Python $PYTHON_VERSION applications and frameworks. \
+Python is an easy to learn, powerful programming language. It has efficient high-level \
+data structures and a simple but effective approach to object-oriented programming. \
+Python's elegant syntax and dynamic typing, together with its interpreted nature, \
+make it an ideal language for scripting and rapid application development in many areas \
+on most platforms."
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Python 3.6" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,python,python36,rh-python36" \
+      com.redhat.component="python36-docker" \
+      name="centos/python-36-centos7" \
+      version="3.6" \
```

Comment:
Can we set the version to 1 in the CentOS specfile like it is in the RHEL one? I think we could, since it's a new image and its name already contains the Python version.